.github/workflows/pypi_upload.yml

```diff
@@ -0,0 +1,51 @@
+# Action to check the version of the package and upload it to PyPI
+# if the version is higher than the one on PyPI
+# Author: @quantumstud
+name: PyPI Upload
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check_version:
+    runs-on: ubuntu-latest
+    name: Check if version.py is changed
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get changed files
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v32
+        with:
+          files: |
+            py4DSTEM/version.py
+      - name: Run step if any file(s) in the docs folder change
+        if: steps.changed-files-specific.outputs.any_changed == 'false'
+        run: |
+          echo "Version file not changed, run script to change the version file."
```

Comment:
suggest changing to "running" rather than "run" - this happens automatically, but "run" sounds like the user needs to do something

Reply:
Hi Ben,
This is a restricted command for the GitHub workflow, which we cannot change. This instructs the workflow to run a specific set of instructions while executing the Github actions.